recipes/soci/all/conanfile.py

```diff
@@ -0,0 +1,154 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class SociConan(ConanFile):
+    name = "soci"
+    homepage = "https://github.com/SOCI/soci"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "The C++ Database Access Library "
+    topics = ("C++", "database-library", "oracle", "postgresql", "mysql", "odbc", "db2", "firebird", "sqlite3", "boost" )
+    license = "Boost"
```

Comment:
```suggestion
    license = "BSL-1.0"
```

Use official short name: https://spdx.org/licenses/BSL-1.0.html